default_lcf.xml

```diff
@@ -56,16 +56,20 @@
 
         <!-- R File Appender  set to output to a single log file.
             This is defined for systems that can't (or don't want to) have rolling files. -->
-        <File name="R" fileName="${sys:jmri.log.path}session.log" append="false">
+        <RollingFile name="R" fileName="${sys:jmri.log.path}session.log"
```

Comment:
This looks a bit strange to me, but it might be correct.
The comment says that "R" is for output to a single file, but the tag is `RollingFile`. Just checking that this generates only one file and doesn't do rolling files.

Reply:
No harm in checking on this one.
So the File can roll when it reaches a maximum size, it's now a RollingFile.
Adding the SizeBasedTriggeringPolicy to a normal File Appender will generate Log4J2 config. errors.